tests/cypress/latest/e2e/unit_tests/seed_image.spec.ts

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright © 2022 - 2023 SUSE LLC
```

Comment:
New file, should be more `Copyright © 2023 SUSE LLC` I think (and this reminds me that I may have did the same mistake :D).

@kkaempf any thoughts on this?